lab/admin/project.py

```diff
@@ -102,6 +104,16 @@ class BeamTimeRequestInline(LabPermissionMixin, admin.StackedInline):
     def get_related_project(self, obj: Optional[Project] = None) -> Optional[Project]:
         return obj
 
+    def formfield_for_dbfield(
+        self, db_field: Field, request: Optional[HttpRequest], **kwargs: Any
+    ) -> Optional[Field]:
+        if (
+            isinstance(db_field, models.TextField)
+            and db_field.name == "problem_statement"
+        ):
+            return CharField(max_length=db_field.max_length, widget=CounterTextarea())
```

Comment:
Si tu remplaces le `CharField` ici, il faut aussi ajouter `label` et `help_text` car ils ne sont plus générés automatiquement par le `ModelFom`.
(cf. la note de cette section : https://docs.djangoproject.com/en/4.0/topics/forms/modelforms/#overriding-the-default-fields).

Tu peux aussi créer un `ModelForm` et juste définir le widget dans la classe `Meta` ?

Reply:
Et pour le coup il y a un `help_text` sur ce champ

Reply:
Oui tu as raison pour le `ModelForm`, c'est mieux.